crates/pipeline-manager/src/db/types/pipeline.rs

```diff
@@ -263,6 +263,7 @@ pub struct ExtendedPipelineDescrMonitoring {
     pub created_at: DateTime<Utc>,
     pub version: Version,
     pub platform_version: String,
+    pub program_config: serde_json::Value,
```

Comment:
more likely to cause backward compat trouble in the future if it isnt strongly typed

Reply:
This would be independent from the current PR; I only added this field to the abbreviated descriptor. It's already part of the full descriptor.